guide/14-deep-learning/how-sequencetosequence-works.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p><s>method accepts one of the </s><code><s>supported_backbones</s></code> - method accepts a backbone name (one of the values returned by call to <code>SequenceToSequence.supported_backbones</code> ) as an argument </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/869/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Reply:
<p>Need discussion.</p>

Reply:
Resolved post discussion without any modification.